infra/experimental/sanitizers/ExecSan/Makefile

```diff
@@ -14,4 +14,4 @@ test:  all vuln.dict
 	./execSan ./target -dict=vuln.dict
 
 clean:
-	rm -f execSan /tmp/tripwire target 
+	rm -f execSan /tmp/tripwire target
```

Comment:
Do we need to rm /tmp/tripwire anymore?

Reply:
I think we do, because [it is created inside `execSan.cpp`](https://github.com/google/oss-fuzz/blob/a04611d13afedef547d1fa7dcf392188c979d225/infra/experimental/sanitizers/ExecSan/execSan.cpp#L393) 😀.